code/style.css

```diff
@@ -1,4 +1,317 @@
+
+/* RESET CSS TO BORDER-BOX */
+* {
+  box-sizing: border-box;
 }
+
+body {
+  background-color: #e6e6e6;
+  margin: 0;
+}
+
+header {
+  background-color: #994C41;
+  padding: 1rem;
+}
+
+.logo {
+  color: #dcd1b4;
+  font-family: 'Playfair Display';
+  font-size: 1rem;
+}
+
+h1, h2, h3 {
+  color: #dcd1b4;
+  font-family: 'Playfair Display';
+  font-size: 8vw;
+  font-style: normal;
+  font-weight: normal;
+  text-align: center;
+}
+
+h4 {
+  margin: 1rem 0;
+  font-family: 'Playfair Display';
+  font-size: 1.5rem;
+  font-style: normal;
+  font-weight: normal;
+  text-transform: uppercase;
+}
+
+a {
+  text-decoration: none;
+  color: #dcd1b4;
+  font-family: 'poppins'; 
+  font-size: 1rem;
+}
+
+p {
+  font-family: 'poppins'; 
+  color: #dcd1b4;
+  font-size: 1rem;
+}
+
+nav {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin-bottom: 3rem;
+}
+
+nav a, p { 
+  margin: 0;
+}
+
+.centered-font {
+  text-align: center;
+}
+
+.hamburger-menu {
+  display: flex;
+  flex-direction: column;
+}
+
+.menu-line {
+  width: 1.5rem;
+  height: 1px;
+  margin-bottom: 0.5rem;
+  fill: #dcd1b4;
+  background-color: #dcd1b4;
+}
+
+.desktop-menu {
+  display: none;
+  text-transform: uppercase;
+}
+
+.desktop-menu a {
+  margin-left: 1.5rem;
+}
+
+.desktop-menu a:hover {
+  color: #994C41;
+  background-color: #dcd1b4;
+  text-decoration: none;
+}
+
+.hero-image {
+  width: 100%;
+}
+
+.hero-description {
+  margin-bottom: 1.5rem;
+}
+
+.hero-description h4 {
+  color: #dcd1b4;
+}
+
+.read-more { 
+  font-family: 'poppins'; 
+  color: #dcd1b4;
+  font-size: 1rem;
+  text-decoration: underline;
+  text-transform: uppercase;
+}
+
+.read-more:hover {
+  color: #994C41;
+  background-color: #dcd1b4;
+  text-decoration: none;
+}
+
+main {
+  padding: 1rem;
+}
+
+.home-intro h2 {
+  color: #212121;
+  font-size: 5.9vw;
+  margin: 3rem;
+}
+
+.article-header {
+  margin: 0 0 0 1rem;
+  color: #212121;
+  font-family: poppins;
+  text-align: left;
+}
+
+.article-link {
+  text-decoration: underline;
+}
+
+.article-link:hover {
+  color: #e6e6e6;
+  background-color: #212121;
+  text-decoration: none;
+}
+
+.article-section {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  flex-wrap: wrap;
+}
+
+.article {
+  width: calc(100% - 2rem);
+  margin: 1rem;
+  padding: 1rem;
+}
+
+.yellow {
+  background-color: #816d3b;
+  color: #cad4dc;
+}
+
+.yellow p {
+  color: #cad4dc;
+}
+
+.blue {
+  background-color: #65798e;
+  color: #672f2a;
+}
+
+.blue p {
+  color: #672f2a;
+}
+
+.plum {
+  background-color: #5a4041;
+  color:#dcd1b4;
+}
+
+.plum p {
+  color:#dcd1b4;
+}
+
+.darkblue {
+  background-color: #243546;
+  color: #a8925e;
+}
+
+.darkblue p {
+  color: #a8925e;
+}
+
+.darkgreen {
+  background-color: #424635;
+  color:#dcd1b4;
+}
+
+.darkgreen p {
+  color:#dcd1b4;
+}
+
+.beige {
+  background-color: #dcd1b4;
+  color: #672f2a;
+}
+
+.beige p {
+  color: #672f2a;
+}
```

Comment:
The class of the colors (.yellow, .blue, .plum...) for example is a little hard to understand what it does. Does it change the color of the article? the card maybe?